client/landing/stepper/declarative-flow/internals/steps-repository/plans/style.scss

```diff
@@ -199,4 +199,9 @@
 	#step-header {
 		margin: 0;
 	}
+
+	.plan-features-2023-grid__popular-badge.is-current-plan,
+	.plan-features-2023-grid__popular-badge.is-business-plan {
+		display: none;
+	}
```

Comment:
Should we be more specific and add `.start-writing` as the root element here?

Reply:
GH preview didn't show, but we're inside `.start-writing` here. Is that what you meant?

![image](https://github.com/Automattic/wp-calypso/assets/1044309/d3f47d51-b0e3-472c-8d21-17046fe7fff1)

Reply:
Haha! So we are! 😅 LGTM!